rules/windows/file/file_event/file_event_win_ps_script_policy_test_creation_by_uncommon_process.yml

```diff
@@ -15,19 +15,25 @@ logsource:
 detection:
     selection:
         TargetFilename|contains: '__PSScriptPolicyTest_'
+    filter_main_powershell:
+        - Image|endswith:
+              - 'C:\Program Files\PowerShell\7-preview\pwsh.exe'
+              - 'C:\Program Files\PowerShell\7\pwsh.exe'
+              - 'C:\Windows\System32\WindowsPowerShell\v1.0\powershell_ise.exe'
+              - 'C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe'
+              - 'C:\Windows\SysWOW64\WindowsPowerShell\v1.0\powershell_ise.exe'
+              - 'C:\Windows\SysWOW64\WindowsPowerShell\v1.0\powershell.exe'
+        - Image|contains:
+              - 'C:\Program Files\WindowsApps\Microsoft.PowerShellPreview'
+              - '\AppData\Local\Microsoft\WindowsApps\Microsoft.PowerShellPreview'
+          Image|endswith: '\pwsh.exe'
```

Comment:
please don't mix and/or YAML grouping. For readability this should be its own selection.